mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution/ContentView.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+import SwiftUI
+
+struct ContentView: View {
+    @State private var performSuperRes = false
+    
+    func runOrtSuperResolution() -> UIImage? {
+        do {
+            let outputImage = try ORTSuperResolutionPerformer.performSuperResolution()
+            return outputImage
+        } catch let error as NSError {
+            print("Error: \(error.localizedDescription)")
```

Comment:
is it easy to see the error message or should we show it in the UI?

Reply:
I guess all the print message in swift app will shown in the console output area. looks like it's common to use try/catch print(error message) in swift error handling.

currently the UI will show `unable to perform super resolution,` the specific error message should be easy to see in the console output area. but we can also consider to add it to UI of course.